### DISCUSSION TITLE
How will we know when new version of API is released (containing bug fix for "duplicate" alerts)?

### DISCUSSION THREAD

**Discussion by ckolovson** [weather-gov/api#681] [Q&A]
How will we know when the bug fix that filters out "duplicate" alerts will be released? I'd like to know when to switch from using preview-api.weather.gov to api.weather.gov.

**Comment by sullynole** [comment]
Operational bug reports are normally tracked in tickets to nco.ops@noaa.gov and they will update the customers on the ticket when this issue is resolved, planned for this coming Wednesday.  I would not recommend using preview-api.weather.gov in the interim as that is used for testing and could be missing data or go down at any time.
